www/HKDC.svg

```diff
@@ -0,0 +1,120 @@
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:133.33332825px;line-height:2;font-family:'Segoe UI';-inkscape-font-specification:'Segoe UI Bold';stroke-width:0.75131482"
+         id="flowPara847" /></flowRoot>    <flowRoot
+       xml:space="preserve"
+       id="flowRoot815-2-0"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:133.33332825px;line-height:2;font-family:'Segoe UI';-inkscape-font-specification:'Segoe UI';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.75131482"
+       transform="matrix(0.35216042,0,0,0.35216042,71.817955,201.53644)"><flowRegion
+         id="flowRegion817-6-5"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:133.33332825px;line-height:2;font-family:'Segoe UI';-inkscape-font-specification:'Segoe UI';stroke-width:0.75131482"><rect
+           id="rect819-4-6"
+           width="985.71429"
+           height="422.85715"
+           x="45.714287"
+           y="75.500565"
+           style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:133.33332825px;line-height:2;font-family:'Segoe UI';-inkscape-font-specification:'Segoe UI';stroke-width:0.75131482" /></flowRegion><flowPara
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:133.33332825px;line-height:2;font-family:'Segoe UI';-inkscape-font-specification:'Segoe UI Bold';stroke-width:0.75131482"
+         id="flowPara847-6">INFO</flowPara></flowRoot>    <flowRoot
+       xml:space="preserve"
+       id="flowRoot878"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:40px;line-height:1.25;font-family:'Cambria Math';-inkscape-font-specification:'Cambria Math';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.6830135"
+       transform="matrix(0.38737645,0,0,0.38737645,-3.3575162,114.80844)"><flowRegion
+         id="flowRegion880"
+         style="stroke-width:0.6830135"><rect
+           id="rect882"
+           width="288.57144"
+           height="431.42856"
+           x="40"
+           y="86.929138"
+           style="stroke-width:0.6830135" /></flowRegion><flowPara
+         id="flowPara884"
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:160px;font-family:'Microsoft YaHei';-inkscape-font-specification:'Microsoft YaHei Bold';stroke-width:0.6830135">香</flowPara><flowPara
+         id="flowPara886"
+         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:160px;font-family:'Microsoft YaHei';-inkscape-font-specification:'Microsoft YaHei Bold';stroke-width:0.6830135">港</flowPara></flowRoot>  </g>
+</svg>
```

Comment:
Do we need the `.svg` file? I can only see the `.png` file being used.

Reply:
It was used to create the .png image - it could be useful if we have future applications / versions of the logo? Could remove it if we want to keep the repo 'minimal', no strong preference either way!

Reply:
Let's keep it then!